test/node-labels.test.js

```diff
@@ -147,3 +147,60 @@ tap.test('label: "lib / src" when more than 5 sub-systems has been changed', (t)
 
   t.end()
 })
+
+tap.test('label: "meta" when meta-info files have changed', (t) => {
+  // e.g. LICENSE, AUTHORS, some ./*.md files
+  const labels = nodeLabels.resolveLabels([
+    '.gitattributes',
+    '.gitignore',
+    '.mailmap',
+    'AUTHORS',
+    'LICENSE',
+    'CHANGELOG.md',
+    'CODE_OF_CONDUCT.md',
+    'GOVERNANCE.md',
+    'ROADMAP.md',
+    'WORKING_GROUPS.md'
+  ])
+
+  t.same(labels, ['meta'])
+
+  t.end()
+})
+
+tap.test('label: not "meta" when other top-level have been changed', (t) => {
+  const labels = nodeLabels.resolveLabels([
+    'BUILDING.md',
+    'README.md',
+    'COLLABORATOR_GUIDE.md',
+    'CONTRIBUTING.md',
+    'configure'
+  ])
+
+  t.same(labels.meta, undefined)
```

Comment:
Hmm... As `labels` is an array `labels.meta` would always be `undefined`. Probably replace this with `Array.indexOf()` or `Array.includes()` if Travis CI has v6 available (?).